website/src/docs/companion.md

```diff
@@ -85,6 +85,9 @@ const options = {
 }
 
 app.use('/companion', companion.app(options))
+
+// Optional: if you want Prometheus metrics:
+app.use(companion.metrics())
```

Comment:
Would it make sense to expose a `metrics: true` option in `companion.app({})` instead? I imagine you would normally want to mount the metrics endpoint on `/companion/` anyways

Reply:
I thought about that, but a couple of issues:
- I believe `/metrics` is currently not under `/companion/`. It would be a breaking change to move it, no?
- It may give less flexibility as to where the user registers their metrics endpoint, as only request handlers/middleware registered *after* prometheus will be recorded, which can be used to ignore certain endpoints from metrics.

Reply:
What I mean is if users mount companion on `/companion`, the metrics endpoint should probably be on `/companion/metrics`. It kinda has to be else we will collect metrics for unrelated endpoints too(?)
In the standalone server we mount `companion.app()` at the root, so the endpoint would still be `/metrics`.

Reply:
Isn't it mounted at `COMPANION_PATH` in standalone?
https://github.com/transloadit/uppy/blob/cd3b3967dacecf55eec6fbbb229c5d8879ee4df3/packages/%40uppy/companion/src/standalone/index.js#L172

Reply:
Aaaa OK I see, yes then it would be a change in behaviour. I don't think that was a conscious choice, but a mistake.

Maybe the standalone server can continue to use middleware function as it does in this PR to prevent breaking changes, while the public API for end users would be using a `metrics: true` option? I can't really think of a reason why you would ever want to mount it elsewhere

Reply:
Yes, can do that

Reply:
Let me know if it looks good now